subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/CodeNarc.java

```diff
@@ -63,6 +67,7 @@
 
     public CodeNarc() {
         reports = getInstantiator().newInstance(CodeNarcReportsImpl.class, this);
+        compilationClasspath = new SimpleFileCollection();
```

Comment:
This should probably rather be `getProject().files()` to avoid a reference to an internal API.